back/spec/services/project_copy_service_spec.rb

```diff
@@ -124,6 +124,63 @@
       expect(template['models']['idea'].first['custom_field_values']).to match expected_custom_field_values
     end
 
+    it 'includes volunteers' do
+      cause = create(:cause)
+      volunteer = create(:user)
+      cause.volunteers.create!(user: volunteer)
+
+      template = service.export cause.participation_context.project, anonymize_users: false, include_ideas: true
+
+      expect(template['models']['volunteering/volunteer'].size).to eq 1
+      expect(template['models']['volunteering/volunteer'].first).to match({
+        'created_at' => an_instance_of(String),
+        'updated_at' => an_instance_of(String),
+        'cause_ref' => hash_including(
+          'title_multiloc' => cause.title_multiloc,
+          'description_multiloc' => cause.description_multiloc,
+          'ordering' => cause.ordering
+        ),
+        'user_ref' => hash_including(
+          'first_name' => volunteer.first_name,
+          'last_name' => volunteer.last_name,
+          'email' => volunteer.email
+        )
+      })
+    end
+
+    it 'includes events' do
```

Comment:
Do we need to add event_image in here?

Reply:
In general I feel like we should rethink how we test templates. The test coverage is extremely low. I skipped images and files because I don't see it being covered anywhere, I think this is because of high effort and that the test is not representative of the production environment (where S3 is used).

Reply:
It would probably also make the specs slow.

Reply:
I added a discussion point to the CDM, I'll merge it for now if that's okay